imports/startup/server/index.coffee

```diff
@@ -313,27 +313,24 @@ api.addRoute 'frequentAgents',
 @apiGroup descriptors
 @apiParam {String} term Infectious Agent
 ###
-api.addRoute 'historicalData/:term',
+api.addRoute 'historicalData/:term/:range',
   get: ->
-    query = prefixes + """
-      SELECT (max(?date) as ?mdt)
-      WHERE {
-        ?phrase anno:category "diseases"
-        ; anno:source_doc ?currentArticle
-        ; anno:selected-text ?rawText
-        ; ^dc:relation ?resolvedTerm
-        .
-        ?resolvedTerm rdfs:label ?termLabel .
-        ?currentArticle pro:post/pro:date ?p_date .
-        OPTIONAL { ?currentArticle  pro:date  ?a_date }
-        BIND(coalesce(?a_date, ?p_date) AS ?date)
-        filter(?termLabel = "#{escape(@urlParams.term)}")
-      }
-      """
-    response = makeRequest(query)
-    recentDate = (response.results.bindings)[0].mdt.value
-    baseYear = moment(recentDate).year() - 5
-
+    dateStr = ""
+    date = moment(new Date())
+    switch @urlParams.range
+      when "6months"
+        date.subtract(6, 'months')
+        dateStr = date.format("YYYY-MM-DD") + "T00:00:00+00:01"
+      when "1year"
+        date.subtract(1, 'years')
+        dateStr = date.format("YYYY-MM-DD") + "T00:00:00+00:01"
+      when "5years"
+        date.subtract(5, 'years')
+        dateStr = date.format("YYYY") + "-01-01T00:00:00+00:01"
+      when "all"
+        date.subtract(100, 'years')
```

Comment:
Why is this offset needed? It looks like the filter isn't used when the date range is "all."